Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -689,4 +689,35 @@ def test_unlink
     cmd("install", "testball")
     assert_match "Would remove", cmd("unlink", "--dry-run", "testball")
   end
+
+  def test_pull_offline
+    assert_match "You meant `git pull --rebase`.", cmd_fail("pull", "--rebase")
+    assert_match "This command requires at least one argument", cmd_fail("pull")
+    assert_match "Not a GitHub pull request or commit",
+      cmd_fail("pull", "0")
+  end
+
+  def test_pull
+    skip if ENV["HOMEBREW_NO_GITHUB_API"]
```

Comment:
Last thing: `skip "Needs online access"` or similar just to provide a bit more information.